tools/release/04_post_publish.ts

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env -S deno run --allow-read --allow-write --allow-run=cargo,git --allow-net --no-check
+// Copyright 2018-2022 the Deno authors. All rights reserved. MIT license.
+import { DenoWorkspace } from "./deno_workspace.ts";
+import { createOctoKit, getGitHubRepository } from "./deps.ts";
+
+const workspace = await DenoWorkspace.load();
+const repo = workspace.repo;
+const cliCrate = workspace.getCliCrate();
+
+console.log("Creating release tag...");
+await createReleaseTag();
+
+console.log("Forwarding release commit to main...");
+try {
+  await forwardReleaseCommitToMain();
+} catch (err) {
+  console.error("Failed. Please manually open a PR.", err);
+}
+
+async function createReleaseTag() {
+  await repo.gitFetchTags("origin");
+  const tags = await repo.getGitTags();
+  const tagName = `v${cliCrate.version}`;
+
+  if (tags.has(tagName)) {
+    console.log(`Tag ${tagName} already exists.`);
+  } else {
+    await repo.gitTag(tagName);
+    await repo.gitPush(tagName);
+  }
+}
+
+async function forwardReleaseCommitToMain() {
```

Comment:
This function is new. Please review it carefully as it's untested, though won't be the end of the world if it fails because it's the last step.